docs/IE_PLUGIN_DG/InferRequest.md

```diff
@@ -1,76 +1,65 @@
+# Synchronous Inference Request {#openvino_docs_ov_plugin_dg_infer_request}
 
 `InferRequest` class functionality:
 - Allocate input and output blobs needed for a backend-dependent network inference.
 - Define functions for inference process stages (for example, `preprocess`, `upload`, `infer`, `download`, `postprocess`). These functions can later be used to define an execution pipeline during [Asynchronous Inference Request](@ref openvino_docs_ie_plugin_dg_async_infer_request) implementation.
 - Call inference stages one by one synchronously.
 
-`InferRequest` Class
+InferRequest Class
 ------------------------
 
-Inference Engine Plugin API provides the helper InferenceEngine::IInferRequestInternal class recommended 
-to use as a base class for a synchronous inference request implementation. Based of that, a declaration 
+OpenVINO Plugin API provides the interface ov::ISyncInferRequest which should be 
+used as a base class for a synchronous inference request implementation. Based of that, a declaration 
 of a synchronous request class can look as follows: 
 
 @snippet src/sync_infer_request.hpp infer_request:header
 
-#### Class Fields
+### Class Fields
 
 The example class has several fields:
 
-- `_executableNetwork` - reference to an executable network instance. From this reference, an inference request instance can take a task executor, use counter for a number of created inference requests, and so on.
-- `_profilingTask` - array of the `std::array<InferenceEngine::ProfilingTask, numOfStages>` type. Defines names for pipeline stages. Used to profile an inference pipeline execution with the Intel® instrumentation and tracing technology (ITT).
-- `_durations` - array of durations of each pipeline stage.
-- `_networkInputBlobs` - input blob map.
-- `_networkOutputBlobs` - output blob map.
-- `_parameters` - `ngraph::Function` parameter operations.
-- `_results` - `ngraph::Function` result operations.
+- `m_profiling_task` - array of the `std::array<openvino::itt::handle_t, numOfStages>` type. Defines names for pipeline stages. Used to profile an inference pipeline execution with the Intel® instrumentation and tracing technology (ITT).
+- `m_durations` - array of durations of each pipeline stage.
 - backend specific fields:
-	- `_inputTensors` - inputs tensors which wrap `_networkInputBlobs` blobs. They are used as inputs to backend `_executable` computational graph.
-	- `_outputTensors` - output tensors which wrap `_networkOutputBlobs` blobs. They are used as outputs from backend `_executable` computational graph.
-	- `_executable` - an executable object / backend computational graph.
+    - `m_backend_input_tensors` - input backend tensors.
+    - `m_backend_output_tensors` - output backend tensors.
+	- `m_executable` - an executable object / backend computational graph.
 
-### `InferRequest` Constructor
+### InferRequest Constructor
 
 The constructor initializes helper fields and calls methods which allocate blobs:
```

Comment:
blobs => tensors.

Can we state more about dynamic shapes case?
E.g. write something like "In case of static shapes, we allocate tensors in ctor, while In dynamic shapes this stage in moved to set_tensor case for inputs, actual infer and post-process for outputs"

Maybe this case be written directly in comments in the code.